aws-predictions/src/main/java/com/amplifyframework/predictions/aws/AWSPredictionsPluginConfiguration.java

```diff
@@ -100,13 +120,57 @@ public NetworkPolicy getDefaultNetworkPolicy() {
         return defaultNetworkPolicy;
     }
 
+    /**
+     * Gets the configuration for text translation.
+     * Null if not configured.
+     * @return the configuration for text translation
+     * @throws PredictionsException if not configured
+     */
+    @NonNull
+    public TranslateTextConfiguration getTranslateTextConfiguration() throws PredictionsException {
```

Comment:
I decided against default configuration for now. Adding each resource via CLI is required to add the appropriate IAM policies, so not having configuration for it would give access denied error down the line anyways.

Reply:
`hasConfiguration` or `isConfigured` sounds useful. I'll add them in.

Reply:
Oops sorry I think we already discussed this in an earlier PR, my b.